docs/contributing.md

```diff
@@ -23,7 +23,7 @@ Please be sure to follow the simple rules:
 
 Make a [fork of this repo](https://github.com/crweiner/hacktoberfest-swag-list/fork) and add the details for what company and swag you find in the [list.md](/docs/list.md) file located at /docs/list.md.
 
-We are using a very simple language called Markdown to format this list. It's basically a way to make things look pretty without having to use a rich text editor. Please familiarize yourself with Markdown [using this handy cheat sheet provided by GitHub](https://guides.github.com/pdfs/markdown-cheatsheet-online.pdf)
+We are using a very simple language called Markdown to format this list. It's basically a way to make things look pretty without having to use a rich text editor. Please familiarize yourself with Markdown [using this handy cheat sheet provided by GitHub](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax)
```

Comment:
Sad to see they got rid of the Markdown Cheatsheet PDF. That simple 2 page sheet was a staple of the hackathons I used to lead (pre-pandemic).

Thank you for finding their new, full guide. It is very detailed and through! I think for _this_ particular project, because it draws in so many brand new people, sending them to a very detailed MD guide may be a bit overwhelming. To keep things simple like the old guide was, I'll swap this out for the new PDF MD guide you also  found. :+1:

Thank you for being on top of this and providing a few different options!